package.json

```diff
@@ -21,16 +21,30 @@
     "url": "https://github.com/stoplightio/spectral"
   },
   "license": "Apache-2.0",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "src/index.ts",
+  "bin": {
+    "spectral": "./bin/run"
+  },
+  "oclif": {
+    "commands": "./dist/cli/commands",
+    "bin": "spectral",
+    "plugins": [
+      "@oclif/plugin-help"
+    ]
+  },
   "files": [
-    "**/*"
+    "./*",
+    "src/*",
+    "src/functions",
+    "src/rulesets",
+    "src/__tests__",
```

Comment:
This probably is not needed - the way `sl-scripts build` works is that all the implementation files are built and put in `dist` folder. Then we cd into that folder and publish. Hence the `**/*`, since all the files in the folder we're publishing from are applicable (dist). Basically `yarn release` looks like this:

```
- build, which compiles all implementation files in src to `dist` (ignoring tests etc)
- cp package.json dist/package.json
- run script to update a couple of things in dist/package.json
- yarn publish
```

So since it's publishing from dist (which makes for nice imports that don't have `dist` in them e.g. `import { resolver } from '@stoplight/spectral/resolver'`) the `**/*` is what we want, which includes everything in the folder we're publishing from (dist).

Reply:
You are right. Forgot we use `sl-scripts` here.